APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.h

```diff
@@ -55,6 +55,15 @@
                  onCompletion:(void (^)(NSError *))completionBlock;
 
 - (void) signInOnCompletion:(void (^)(NSError * error))completionBlock;
+
+/**
+ * Login a user on this device via externalId where registration was handled on a different device
+ * ExternalID property must not be nil before calling this method
+ * @param defaultEmailAddress is the email address that will be used as the basis for creating the external ID login
```

Comment:
I'd call it baseEmailAddress, and document that it shouldn't include any "+" before the .

Better still, I'd follow the BridgeAppSDK pattern of putting the base email address in a configuration parameter (we use a .plist, could be JSON or whatever) and not explicitly passing it in, because (a) it will be fixed per study, but could be different between debug and prod builds or whatever, (b) it's easier to find stuff like that to change if it's in a central location, and (c) you don't just want people calling this willy-nilly with random email addresses because it won't work.